ecc/tests/test_pages.py

```diff
@@ -0,0 +1,45 @@
+from pytest import mark
+
+from django.shortcuts import reverse
+from django.test import override_settings
+
+from tests import factories
+
+
+pytestmark = mark.django_db
+
+
```

Comment:
ajouter un test : si DEMO_INSPECTOR_USERNAME is not set, ca permet pas de se login

Reply:
Je n'ai pas réussi à rajouté ce test. J'ai fait un test manuel.